mmc-google-analytics/src/main/java/info/novatec/metricscollector/google/RequestBuilder.java

```diff
@@ -103,39 +97,27 @@ public RequestBuilder addDimensionFilters(String dimensionName, DimensionFilterO
     }
 
     /**
-     * Defines the date range in which metrics should be collected.
-     * In case this method is not invoked, a default value for yesterday's data is used for collection.
-     *
-     * @param startDate the begin of the period as String
-     * @param endDate the end of the period as String
-     */
-    private RequestBuilder addDateRange(LocalDate startDate, LocalDate endDate) {
-        String pattern = "yyyy-MM-dd";
-        dateRange = new DateRange();
-        dateRange.setStartDate(startDate.format(ofPattern(pattern)));
-        dateRange.setEndDate(endDate.format(ofPattern(pattern)));
-        return this;
-    }
-
-    /**
-     * This method builds the body of the request to be sent
+     * This method builds the body of the request to be sent.
      *
-     * @return
+     * @param viewId the viewId for the specific module that is being requested, fo example: AqeBlog.
```

Comment:
e.g. at the end is a bit confusing. "AqeBlog" can be understood as an example for viewId. Suggestion:
@param viewId for the specific module(e.g. AqeBlog) that is being requested

If you think the example is well placed, then please correct to "for example"